tests/integration/test_vulnerability_detector/test_feeds/test_missing_fields_redhat_feeds.py

```diff
@@ -23,7 +23,7 @@
 # Variables
 current_test_path = os.path.dirname(os.path.realpath(__file__))
 test_data_path = os.path.join(current_test_path, 'data')
-configurations_path = os.path.join(test_data_path, 'configuration', 'wazuh_missing_fields_redhat_feeds.yaml')
+configurations_path = os.path.join(test_data_path, 'configuration', 'wazuh_invalid_redhat_feeds.yaml')
```

Comment:
For consistency with other `path` variables, this should be in a variable.
```suggestion
configurations_path = os.path.join(test_data_path, 'configuration', INVALID_RHEL_FEEDS)
```

Reply:
Done in ab1c229f9625689d173018ae41d5437228ded6be